### ISSUE TITLE
Please update dll.

### ISSUE THREAD

**Issue by ProbablyWorth** [ArranzCNL/ImprovedCameraSE#122]
Please update the DLL to the newest game version of 1.6.640. No rush.

**Comment by KuroYasha9** [comment]
How do I install this mod lol?

**Comment by SerkelDays** [comment]
Everyone would hope that he would have updated it a couple of weeks ago, but it seems that all that trace magically disappeared...

**Comment by killzoms** [comment]
Duplicate of #42

**Comment by ProbablyWorth** [comment]
> [KuroYasha9](/KuroYasha9)

Download the zip, then use your mod manager's non-nexus installer on that zip and it will install

**Comment by JSabelli-FDX** [comment]
Yes please.

**Comment by ArranzCNL** [comment]
Beta6 is available on [discord link](https://discord.com/invite/TqDzcWQmgU) for all SkyrimSE versions under updates.
Beta7 should be available this upcoming weekend, after that it is being released on Nexus.

**Comment by vee2099** [comment]
@ArranzCNL has the discord link expired? It's coming up invalid for me, and I've tried on different machines as well as with [this tool](https://toolscord.com/invite), and both logged in or not. I saw in the other thread someone said it was invalid for them, but you said it was working for you, just wondering if this time it actually expired, or if there's anything else I can try. I'd really like to get the 1.6.640 Improved Camera, I loved it before I updated. Thanks
